astro-migration-provider/README.rst

```diff
@@ -0,0 +1,112 @@
+Astronomer Migration Provider
+=============================
+
+Apache Airflow Provider containing Operators from Astronomer. The purpose of these operators is to better assist customers migrating to Astronomer hosted Airflow environments from MWAA, GCC, OSS. This provider is meant for MWAA 2.0.2 and Composer 1 since the plugin methods are unavailable.
+
+Installation
+------------
+
+Install and update using `pip <https://pip.pypa.io/en/stable/getting-started/>`_:
+
+.. code-block:: bash
+
+    pip install astronomer-migration-provider
+
+
+
+Starship
+========
+
+Starship is an Astronomer utility that assist end users with migrating Variables, Connections, and Environment Variables from their source Airflow environments to an Astronomer environment.
+
+Usage
+-----
+1. Add the following line to your ``requirements.txt`` in your source environment:
+
+.. code-block:: bash
+
+    astronomer-migration-provider
+
+2. Add the following DAG to your source environment:
+
+.. code-block:: python
+
+   from airflow import DAG
+
+   from astronomer.migration.operators import AstroMigrationOperator
+   from datetime import datetime
+
+   with DAG(
+      dag_id="astronomer_migration_dag",
+      start_date=datetime(2020, 8, 15),
+      schedule_interval=None,
+   ) as dag:
+
+      AstroMigrationOperator(
+          task_id='export_meta',
+          deployment_url='{{ var.value.deployment_url }}',
+          token='{{ var.value.astro_token }}',
```

Comment:
```suggestion
          deployment_url='{{ dag_run.conf["deployment_url"] }}',
          token='{{  dag_run.conf["astro_token"] }}',
```

My suggestion is still to utilize the `dag_run.conf`.
- fewer setup steps
- fewer changes to the customer environment
- harder to set things wrong